packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -291,18 +291,16 @@ export default class ImageEdit implements EditorPlugin {
             this.clearDndHelpers();
 
             // Apply the changes, and add undo snapshot if necessary
-            if (
-                applyChange(this.editor, this.image, this.editInfo, this.lastSrc, this.wasResized)
-            ) {
-                this.editor.addUndoSnapshot(() => this.image, ChangeSource.ImageResize);
-            }
+            applyChange(this.editor, this.image, this.editInfo, this.lastSrc, this.wasResized);
```

Comment:
why do we need to trigger ContentChange when we set and remove the wrapper? When we use insertEntity, we do trigger ContentChangeEvent. Do will need to this for another wrappers?

Reply:
Actually this is not for the wrapper. We trigger the event here because it is possible image got resized before this point. So content is really changed.

Reply:
If you ask if that is "faster", the answer is yes.
But doing this is not for performance, but just want to make it simpler. We can remember the image size before resize and check it. But that will introduce additional complexity. And even user click the image but didn't really resize it, we can still trigger this event and nothing harmful.